js/core/utils.js

```diff
@@ -34,13 +34,35 @@ define(
                 return function(doc, insertionPoint) {
                     node.remove();
                     doc.adoptNode(node);
-                    if (insertionPoint.firstElementChild) {
```

Comment:
If `insertionPoint` is never a `Document` object, then this is unnecessary, as all the ElementTraversal methods are supported on Element types in Edge :-)